exporters/otlpjson/README.md

```diff
@@ -0,0 +1,9 @@
+# OpenTelemetry - OTLP JSON Exporters
```

Comment:
If this is purely a logging exporter, this module should be named `logging-otlp` I think.

Reply:
Yeah - there's some tension with the logging signal. But went with that name.

Reply:
heh...yeah, I had forgotten about that bit.

Reply:
oh no, I hadn't thought about that either, that's going to be very confusing, should we rename our existing logging exporter to "file" exporter, or "console" exporter, or ...